devito/ir/clusters/cluster.py

```diff
@@ -69,6 +70,26 @@ def unknown(self):
     def tensors(self):
         return self.trace.tensors
 
+    @property
+    def dtype(self):
+        """Return the arithmetic data type of this Cluster. If the Cluster is
+        performing floating point arithmetic, then any equation performing
+        integer arithmetic is ignored, assuming that they are only carrying
+        out array index calculations. If two equations are performing floating
+        point calculations with mixed precision, return the data type with
+        highest precision."""
+        dtypes = {i.dtype for i in self.exprs}
+        fdtypes = {i for i in dtypes if np.issubdtype(i, np.float)}
```

Comment:
`FutureWarning: Conversion of the second argument of issubdtype from `float` to `np.floating` is deprecated. In future, it will be treated as `np.float64 == np.dtype(float).type`.

This warning pollutes the log quite a lot.

Reply:
this warning must disappear. Can you reproduce it on your laptop/workstation? I cannot on my machines...

Reply:
right, this was due to NumPy 14, ,while I had 13.3 on my machine. Now it should be fixed